corehq/apps/reports/standard/project_health.py

```diff
@@ -48,49 +49,81 @@ class MonthlyPerformanceSummary(jsonobject.JsonObject):
+        self._users_filtered_by_group = users_filtered_by_group
         self._next_summary = None
         self._base_queryset = MALTRow.objects.filter(
             domain_name=domain,
             month=month,
             user_type__in=['CommCareUser', 'CommCareUser-Deleted'],
         )
+        self._apply_group_filter = self._base_queryset.filter(
+            user_id__in=users_filtered_by_group,
+        )
         self._performing_queryset = self._base_queryset.filter(
             num_of_forms__gte=performance_threshold,
         )
+        self._performing_queryset_group_filtered = self._apply_group_filter.filter(
+            num_of_forms__gte=performance_threshold,
+        )
         super(MonthlyPerformanceSummary, self).__init__(
             month=month,
             domain=domain,
             performance_threshold=performance_threshold,
-            active=self._base_queryset.distinct('user_id').count(),
-            performing=self._performing_queryset.distinct('user_id').count(),
+            active=0,
+            performing=0,
         )
 
+    def set_performing(self, num_performing_users):
+        self.performing = num_performing_users
+
+    def set_active_users(self, num_active_users):
+        self.active = num_active_users
+
     def set_next_month_summary(self, next_month_summary):
         self._next_summary = next_month_summary
 
     @property
+    def number_of_performing_users(self):
+        if self._users_filtered_by_group:
+            return self._performing_queryset_group_filtered.distinct('user_id').count()
+        else:
+            return self._performing_queryset.distinct('user_id').count()
+
+    @property
+    def number_of_active_users(self):
+        if self._users_filtered_by_group:
+            return self._apply_group_filter.distinct('user_id').count()
+        else:
+            return self._base_queryset.distinct('user_id').count()
+
+    @property
     def previous_month(self):
         prev_year, prev_month = add_months(self.month.year, self.month.month, -1)
         return datetime.datetime(prev_year, prev_month, 1)
 
     @property
     def delta_performing(self):
-        return self.performing - self._previous_summary.performing if self._previous_summary else self.performing
+        if self._previous_summary:
+            return self.number_of_performing_users - self._previous_summary.number_of_performing_users
+        else:
+            return self.number_of_performing_users
 
     @property
     def delta_performing_pct(self):
-        if self.delta_performing and self._previous_summary and self._previous_summary.performing:
-            return float(self.delta_performing / float(self._previous_summary.performing)) * 100.
+        if self._previous_summary.number_of_performing_users:
+            return float(self.delta_performing / float(self._previous_summary.number_of_performing_users)) * 100.
+        else:
+            return 100
```

Comment:
maybe related to the above - this makes sense if `self._previous_summary.number_of_performing_users` is equal to `0`, but if `self._previous_summary` isn't set at all then returning `100` seems a bit misleading.